community/concepts/grain.md

```diff
@@ -0,0 +1,73 @@
+### Grain
+
+SourceCred is both a social algorithm for assigning [cred](concepts/cred.md)
+scores, and a cryptoeconomic system for rewarding a project's contributors
+and sponsors. That cryptoeconomic system revolves around _grain_, a cred-based
+cryptoasset.
+
+The key properties of grain include:
+
+- **Grain is project-specific.**
+
+It's not a universal token; every project that is using SourceCred
+creates its own independent token. They can call it something other
+than grain (e.g. "Seeds") if they want.
+
+- **Grain is harvested by contributors, based on cred scores.**
+
+Grain is created during "harvests". In a harvest, a project mints new grain and
+distributes it to contributors based on their cred scores. How the grain is
+harvested and distributed is up to the project. For example, a project could
+mint a fixed amount of grain every week and distribute it to contributors based
+on their recent contributions, or it could have a target "grain-per-cred"
+payout ratio, and harvest to ensure that all contributors receive at least that
+amount.
+
+- **Grain is transferrable and tradeable.**
+
+Holders of grain can freely send it to others, or exchange it. Within a
+project, grain is [fungible]. If grain has been put on a blockchain (e.g. as an
+[ERC20 token]), then it can be swapped or traded via protocols like [Uniswap]
+or [0x], just like any other ERC20.
+
+[fungible]: https://en.wikipedia.org/wiki/Fungibility
+[ERC20 token]: https://www.ledger.com/academy/crypto/what-are-erc20-tokens
+[Uniswap]: https://uniswap.org/
+[0x]: https://0x.org/
+
+- **Grain is held by contributors and sponsors.**
+
+Grain is initially distributed to contributors, based on their cred scores.
+Anyone who buys grain is either sponsoring contributors directly (by paying
+them for their grain), or is buying out a previous sponsor.
+
+Thus, anyone holding grain has supported the project, either directly through
+contributions, or indirectly through financial sponsorship.
+
+- **Grain can be used for boosting**
+
```

Comment:
Should this "boosting" be in a separate concept file (and linked here?)

Reply:
Yeah, we should add a separate concept file for boosting. How about we merge this one, and I'll write one for boosting (and update this one to link to it / remove overly redundant info). I think there's no harm in having a little overlap, since boosting is quite important to the concept of grain.

Reply:
Sounds good.